streams/src/main/java/org/apache/kafka/streams/kstream/Consumed.java

```diff
@@ -55,9 +56,12 @@ public class Consumed<K, V> implements NamedOperation<Consumed<K, V>> {
     protected Serde<K> keySerde;
     protected Serde<V> valueSerde;
     protected TimestampExtractor timestampExtractor;
-    protected Topology.AutoOffsetReset resetPolicy;
+    protected AutoOffsetReset resetPolicy;
```

Comment:
We need to update `equals` and `hashCode` to consider this new member variable.

Reply:
Thank you, updated this